scripts/genpinctrl/genpinctrl.py

```diff
@@ -444,8 +444,10 @@ def main(cube_path, output):
     mcu_signals = get_mcu_signals(cube_path, gpio_ip_afs)
 
     if os.path.exists(output):
-        shutil.rmtree(output)
-    os.makedirs(output)
+        for dirpath, dirnames, filenames in os.walk(output):
+            # Remove directories, ignore files (README)
+            for dirname in dirnames:
+                shutil.rmtree(os.path.join(dirpath, dirname))
```

Comment:
there should be an else case here creating the output directory in case it doesn't exist. Even though it will never happen now in the repo, it may be needed for other paths.

Reply:
Indeed, I haven't though of other paths, I'll fix that.

Reply:
Fixed